WorldCat/worldcat.json

```diff
@@ -0,0 +1 @@
 Gesamtausgabe. 3,10 = Serie 3: Kirchenmusik ; Bd. 10, Coronation anthems : HWV 259, 258, 260, 261","artist":"by Georg Friedrich Händel; Stephan Blaut; Hans Dieter Clausen; Michael Pacholke; Georg-Friedrich-Händel-Gesellschaft.","publisher":"Kassel [u.a.] : Bärenreiter, 2015.","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":43,"recording":"The singer's musical theatre anthology","artist":"by Richard Walters;","publisher":"Milwaukee, WI : Hal Leonard Corporation, [2016]","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":44,"recording":"Die Walküre","artist":"by Richard Wagner; Frederick Jameson; Alfred Ernst","publisher":"London ; New York : Ernst Eulenburg ; [Milwaukee] : Distributed in North and South America exclusively by Hal Leonard, [2001?]","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":45,"recording":"Requiem, KV 626 : Das von Franz Xaver Süssmayr vervollständigte Requeim in der traditionellen Gestalt = The Requiem, completed by Franz Xaver Süssmayr, in its traditional form","artist":"by Wolfgang Amadeus Mozart; Franz Xaver Süssmayr; Heinz Moehn","publisher":"Kassel : Bärenreiter, [2016] ©1965","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":46,"recording":"Matthäus-Passion = St. Matthew passion : BWV 244","artist":"by Johann Sebastian Bach; Alfred Dürr; Henry S Drinker","publisher":"Kassel : Bärenreiter, 2015, ©1974.","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":47,"recording":"Hymns ancient and modern : for use in the services of the church, with accompanying tunes.","artist":"by George Frideric Handel","publisher":"Norwich : Canterbury Press, [©1924], 1988.","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":48,"recording":"Judas Maccabaeus : an oratorio for soli, chorus and orchestra","artist":"by Joseph Haydn; Marc-Antoine Charpentier; James Rae","publisher":"[Van Nuys, CA] : Kalmus : Alfred Publishing Co., [2006?]","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":49,"recording":"Poco adagio cantabile : (The emperor quartet)","artist":"by Joseph Haydn; Andreas Friesenhagen; Christin Heitmann","publisher":"Vienna : Universal Edition, cop. 2009.","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"},{"recording_number":50,"recording":"Sinfonie in d = Symphony in D minor : Lamentazione, Hob. I:26","artist":"by Johann Sebastian Bach; Werner Breig","publisher":"Kassel ; New York : Bärenreiter, [2015] ©2015","source":"https://www.worldcat.org/search?q=dt%3Asco&fq=&dblist=638&qt=page_number_link&start=1"}]}
```

Comment:
You probably don't want to commit the scraped data to the repo, except maybe for some examples…